unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PlayerName/Tests/PlayerNameShould.cs

```diff
@@ -65,13 +65,13 @@ public void SetNameCorrectly(string name)
     }
 
     [Test]
-    public void DisableTheGameObjectIfHidden()
+    public void DisableRenderersIfHidden()
     {
         playerName.alpha = 0;
         playerName.targetAlpha = 0;
         playerName.Update(float.MaxValue);
 
-        Assert.IsFalse(playerName.gameObject.activeSelf);
+        Assert.IsTrue(playerName.AllRenderersInvisible());
```

Comment:
i dont like the exposure of `AllRenderersInvisible` method just for testing purposes. We are adding production functionality just for testing code. You should be able to check the alpha status of the internal `canvasRenderers` list from this test implementation